Sources/Sentry/include/SentryClient+Private.h

```diff
@@ -19,6 +19,7 @@ NS_ASSUME_NONNULL_BEGIN
 SentryClient (Private)
 
 @property (nonatomic, weak) id<SentryClientAttachmentProcessor> attachmentProcessor;
+@property (nonatomic, strong) SentryThreadInspector *threadInspector;
```

Comment:
`m`: Why do we need this here? Can we also move the `SentryThreadInspector` to the `SentryDependencyContainer`?

Reply:
To instantiate a ThreadInspector we need a bunch of other objects (SentryInAppLogic, SentryCrashStackEntryMapper, SentryStacktraceBuilder, SentryCrashMachineContextWrapper). 

SentryANRTrackingIntegration needs a `ThreadInspector` but I didn't want to create each of this objects again since SentryClient already have this. I guess we could move all of them to the DependencyContainer.

But I don't think we should do it in this PR.